examples/typescript/informer/informer.ts

```diff
@@ -0,0 +1,15 @@
+// tslint:disable:no-console
+import * as k8s from '@kubernetes/client-node';
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
+
+const listFn = () => k8sApi.listPodForAllNamespaces();
+
+const informer = k8s.makeInformer(kc, '/api/v1/namespaces/default/pods', listFn);
```

Comment:
@brendanburns WOW this is really nice! I didn't think promises would make such a huge impact to the API but this looks almost on the same level as the go code! really nice work. :)